src/main/java/de/tum/in/www1/artemis/service/RatingService.java

```diff
@@ -48,6 +48,10 @@ public RatingService(RatingRepository ratingRepository, ResultRepository resultR
         return ratingRepository.findAllByResult_Participation_Exercise_Course_Id(courseId);
     }
 
+    public long countRatingsByCourse(long courseId) {
+        return ratingRepository.countByResult_Participation_Exercise_Course_Id(courseId);
+    }
+
```

Comment:
you could add a doc string here. Even though it's a bit duplicated from the repository documentation. but all the other have documentation as well and also just invoke a repository method.